autoendpoint/src/server/extractors/notification_headers.rs

```diff
@@ -53,17 +79,22 @@ impl NotificationHeaders {
         let ttl = get_header(req, "ttl")
             .and_then(|ttl| ttl.parse().ok())
             // Enforce a maximum TTL, but don't error
-            .map(|ttl| min(ttl, MAX_TTL));
+            .map(|ttl| min(ttl, MAX_TTL))
+            // TODO: Should we return an error if no TTL is given instead of defaulting to 0?
```

Comment:
TTL was introduced after the commonly used draft but before the RFC was released. At this point, I think it's fine to return an error:
https://tools.ietf.org/html/rfc8030#section-5.2